src/csvcubed/models/cube/qb/components/attributevalue.py

```diff
@@ -36,3 +51,23 @@ def get_permitted_rdf_fragment_hints(self) -> Set[RdfSerialisationHint]:
 
     def get_identifier(self) -> str:
         return self.label
+
+    def validate_data(
+        self,
+        data: pd.Series,
+        column_csvw_name: str,
+        csv_column_uri_template: str,
+        column_csv_title: str,
+    ) -> List[ValidationError]:
+        return []
+
+    def _get_validations(self) -> Dict[str, ValidationFunction]:
+
+        return {
+            "label": validate_str_type,
+            "description": validate_optional(validate_str_type),
+            **UriIdentifiable._get_validations(self),
+            "source_uri": validate_optional(validate_uri),
+            "parent_attribute_value_uri": validate_optional(validate_uri),
+            "arbitrary_rdf": validate_list(validate_triple_fragment),
```

Comment:
Can't we get rid of the `validate_triple_fragment` function and replace it with `v.validated_model`?

Reply:
Replaced and TripleFragmentBase now inherits ValidatedModel.